src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -426,6 +426,10 @@ export const drawerNodeItems = ({
               motoscout24: false,
             },
           },
+          entitlementConfig: {
+            hideIfEntitlementIsPresent: Entitlement.CockpitFrontend,
+            singleRequiredEntitlement: [Entitlement.CockpitFrontend],
```

Comment:
This shouldn't be required entitlement?

Reply:
`singleRequiredEntitlement` is the only mandatory props in the config, I could make it optional though but it looks weird